lib/util/support.js

```diff
@@ -152,4 +158,9 @@ function isSupported() {
     && (!isMobile() || supportedMobileBrowsers.includes(browser));
 }
 
-module.exports = isSupported;
+module.exports = {
+  isSupported,
+  // eslint-disable-next-line no-warning-comments
+  // TODO(joyce): Move browser detection to a different util
+  isChromeIOS
```

Comment:
we should expose this as a function instead of variable - that way `guessBrowser` wont get called just be loading twilio-video.